<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Log4J Log Messages</title>
<style type="text/css">
<!--
body, table {font-family: arial,sans-serif; font-size: x-small;}
th {background: #336699; color: #FFFFFF; text-align: left;}
-->
</style>
</head>
<body bgcolor="#FFFFFF" topmargin="6" leftmargin="6">
<hr size="1" noshade>
Log session start time Tue Jun 07 15:34:40 COT 2016<br>
<br>
<table cellspacing="0" cellpadding="4" border="1" bordercolor="#224466" width="100%">
<tr>
<th>Time</th>
<th>Thread</th>
<th>Level</th>
<th>Category</th>
<th>Message</th>
</tr>

<tr>
<td>0</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]</td>
</tr>

<tr>
<td>17</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@229ec9cd, org.springframework.test.context.support.DependencyInjectionTestExecutionListener@66d2c37c, org.springframework.test.context.support.DirtiesContextTestExecutionListener@172036a1, org.springframework.test.context.transaction.TransactionalTestExecutionListener@43e28fe3, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@4f54a2e6]</td>
</tr>

<tr>
<td>113</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>114</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@66d2c37c] to prepare test instance [co.edu.udea.iw.dao.hibernate.PqrDAOHibernateTest@4e205be8]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Log4J Log Messages</title>
<style type="text/css">
<!--
body, table {font-family: arial,sans-serif; font-size: x-small;}
th {background: #336699; color: #FFFFFF; text-align: left;}
-->
</style>
</head>
<body bgcolor="#FFFFFF" topmargin="6" leftmargin="6">
<hr size="1" noshade>
Log session start time Tue Jun 07 15:37:34 COT 2016<br>
<br>
<table cellspacing="0" cellpadding="4" border="1" bordercolor="#224466" width="100%">
<tr>
<th>Time</th>
<th>Thread</th>
<th>Level</th>
<th>Category</th>
<th>Message</th>
</tr>

<tr>
<td>0</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]</td>
</tr>

<tr>
<td>17</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@18329bfc, org.springframework.test.context.support.DependencyInjectionTestExecutionListener@229ec9cd, org.springframework.test.context.support.DirtiesContextTestExecutionListener@66d2c37c, org.springframework.test.context.transaction.TransactionalTestExecutionListener@172036a1, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@43e28fe3]</td>
</tr>

<tr>
<td>110</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>111</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@229ec9cd] to prepare test instance [co.edu.udea.iw.dao.hibernate.PqrDAOHibernateTest@650e2859]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Log4J Log Messages</title>
<style type="text/css">
<!--
body, table {font-family: arial,sans-serif; font-size: x-small;}
th {background: #336699; color: #FFFFFF; text-align: left;}
-->
</style>
</head>
<body bgcolor="#FFFFFF" topmargin="6" leftmargin="6">
<hr size="1" noshade>
Log session start time Tue Jun 07 15:38:54 COT 2016<br>
<br>
<table cellspacing="0" cellpadding="4" border="1" bordercolor="#224466" width="100%">
<tr>
<th>Time</th>
<th>Thread</th>
<th>Level</th>
<th>Category</th>
<th>Message</th>
</tr>

<tr>
<td>0</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]</td>
</tr>

<tr>
<td>18</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@24150b2d, org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6190c9dc, org.springframework.test.context.support.DirtiesContextTestExecutionListener@68b4bc8e, org.springframework.test.context.transaction.TransactionalTestExecutionListener@31c74456, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@115783a2]</td>
</tr>

<tr>
<td>113</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>114</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6190c9dc] to prepare test instance [co.edu.udea.iw.dao.hibernate.UsuarioDAOHibernateTest@177fee56]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>

<tr>
<td>120</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>121</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6190c9dc] to prepare test instance [co.edu.udea.iw.dao.hibernate.UsuarioDAOHibernateTest@5281ce30]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>

<tr>
<td>124</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>124</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6190c9dc] to prepare test instance [co.edu.udea.iw.dao.hibernate.UsuarioDAOHibernateTest@137405e4]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>

<tr>
<td>127</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>127</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@6190c9dc] to prepare test instance [co.edu.udea.iw.dao.hibernate.UsuarioDAOHibernateTest@c222b37]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Log4J Log Messages</title>
<style type="text/css">
<!--
body, table {font-family: arial,sans-serif; font-size: x-small;}
th {background: #336699; color: #FFFFFF; text-align: left;}
-->
</style>
</head>
<body bgcolor="#FFFFFF" topmargin="6" leftmargin="6">
<hr size="1" noshade>
Log session start time Tue Jun 07 15:57:04 COT 2016<br>
<br>
<table cellspacing="0" cellpadding="4" border="1" bordercolor="#224466" width="100%">
<tr>
<th>Time</th>
<th>Thread</th>
<th>Level</th>
<th>Category</th>
<th>Message</th>
</tr>

<tr>
<td>1</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener]</td>
</tr>

<tr>
<td>7</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Could not instantiate TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener]. Specify custom listener classes or make the default listener classes (and their required dependencies) available. Offending class: [javax/servlet/ServletContext]</td>
</tr>

<tr>
<td>18</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.test.context.support.DefaultTestContextBootstrapper category">org.springframework.test.context.support.DefaultTestContextBootstrapper</td>
<td title="Message">Using TestExecutionListeners: [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@2f6c8fd8, org.springframework.test.context.support.DirtiesContextTestExecutionListener@57c63b, org.springframework.test.context.transaction.TransactionalTestExecutionListener@2974baa, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener@1ae8873a]</td>
</tr>

<tr>
<td>188</td>
<td title="main thread">main</td>
<td title="Level">INFO</td>
<td title="org.springframework.beans.factory.xml.XmlBeanDefinitionReader category">org.springframework.beans.factory.xml.XmlBeanDefinitionReader</td>
<td title="Message">Loading XML bean definitions from class path resource [configuracion.xml]</td>
</tr>

<tr>
<td>189</td>
<td title="main thread">main</td>
<td title="Level"><font color="#993300"><strong>ERROR</strong></font></td>
<td title="org.springframework.test.context.TestContextManager category">org.springframework.test.context.TestContextManager</td>
<td title="Message">Caught exception while allowing TestExecutionListener [org.springframework.test.context.support.DependencyInjectionTestExecutionListener@2f6c8fd8] to prepare test instance [co.edu.udea.iw.dao.hibernate.PqrDAOHibernateTest@5d64b55b]</td>
</tr>
<tr><td bgcolor="#993300" style="color:White; font-size : xx-small;" colspan="6">java.lang.IllegalStateException: Failed to load ApplicationContext
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:94)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultTestContext.getApplicationContext(DefaultTestContext.java:72)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.injectDependencies(DependencyInjectionTestExecutionListener.java:117)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.DependencyInjectionTestExecutionListener.prepareTestInstance(DependencyInjectionTestExecutionListener.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:212)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:200)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:259)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:219)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:83)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:163)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: IOException parsing XML document from class path resource [configuracion.xml]; nested exception is java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:344)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:304)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:181)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:217)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:188)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.support.AbstractBeanDefinitionReader.loadBeanDefinitions(AbstractBeanDefinitionReader.java:252)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadBeanDefinitions(AbstractGenericContextLoader.java:253)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:122)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractGenericContextLoader.loadContext(AbstractGenericContextLoader.java:60)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.delegateLoading(AbstractDelegatingSmartContextLoader.java:109)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.support.AbstractDelegatingSmartContextLoader.loadContext(AbstractDelegatingSmartContextLoader.java:261)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:68)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.test.context.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:86)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 25 more
<br>&nbsp;&nbsp;&nbsp;&nbsp;Caused by: java.io.FileNotFoundException: class path resource [configuracion.xml] cannot be opened because it does not exist
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.core.io.ClassPathResource.getInputStream(ClassPathResource.java:172)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	at org.springframework.beans.factory.xml.XmlBeanDefinitionReader.loadBeanDefinitions(XmlBeanDefinitionReader.java:330)
<br>&nbsp;&nbsp;&nbsp;&nbsp;	... 37 more
</td></tr>
